extensions/amp-viewer-integration/0.1/touch-handler.js

```diff
@@ -138,6 +138,10 @@ export class TouchHandler {
    * @private
    */
   forwardEvent_(e) {
+    if (e && e.shouldViewerCancelPropagation) {
```

Comment:
I'd add a comment about conditions this occurs under so it's more obvious in six-months.

Reply:
```suggestion
    if (e?.shouldViewerCancelPropagation) {
```

Reply:
Keep forgetting about this newer syntax. Is there a codemod we could apply on PRs to recommend updated syntax suggestions like this?